snuba/consumer.py

```diff
@@ -110,3 +134,209 @@ def flush_batch(self, batch: Sequence[ProcessedMessage]):
                     )
 
             self.producer.flush()
+
+
+class InsertBatchWriter(ProcessingStep[InsertBatch]):
```

Comment:
`InsertBatch` will likely become generic on the row type soon to differentiate by batches of `InsertBatch[WriterTableRow]` and `InsertBatch[JSONRow]`.